parsl/tests/site_tests/site_config_selector.py

```diff
@@ -73,3 +73,6 @@ def fresh_config():
         # We should skip this.
 
     return config
+
+
+config = fresh_config()
```

Comment:
What is this for?

Reply:
To do `make site_test`, we need this `site_config_selector` to return a `config`, otherwise it returns a `config` not found error

Reply:
looks like this was removed accidentally in #1720